### ISSUE TITLE
System Extension Blocked

### ISSUE THREAD

**Issue by pradhyumk** [chris1111/HoRNDIS#4]
Hello there,
I was attempting to install this on my MacBook Pro and followed all steps, but I repeatedly get an error "System Extension Blocked". As a result, I am unable to approve this in Security & Privacy.

![image](https://user-images.githubusercontent.com/11365510/147464022-c96adc91-7734-4173-a06d-da8fcc89b068.png)

Device: MacBook Pro (13-inch, M1, 2020)
OS: macOS Monterey 12.1

**Comment by chris1111** [comment]
OK M1 Apple Chip
Boot in Recovery HD then  open Terminal and type:
**csrutil enable --without kext**

**Comment by pradhyumk** [comment]
I did that and tried running the PKG again, but it claims installation failed and i did not receive the popup to approve the extension.

**Comment by chris1111** [comment]
> I did that and tried running the PKG again, but it claims installation failed and i did not receive the popup to approve the extension.

Follow exact steps
Do this step after the PKG install

Boot in Recovery HD then open Terminal and type:
csrutil enable --without kext

**Comment by pradhyumk** [comment]
Nope, I did all of the steps all over again from the beginning and added the without kext arg instead of just enable and it still does not work. Throughout this process, I never received the popup to enable the system extension

**Comment by chris1111** [comment]
Read this new PDF
M1 Apple as no Popup 
after, open settings-security, and install button will be added.
Install it and reboot
[Guide.pdf](https://github.com/chris1111/HoRNDIS/files/7780016/Guide.pdf)

**Comment by chris1111** [comment]
I am in work now, tonight when Iam back Home I will make a PKG's specially for M1 Chip

**Comment by chris1111** [comment]
Release [Package for M1 Done](https://github.com/chris1111/HoRNDIS/releases/tag/rel9.2)
See ➤ [install package M1](https://user-images.githubusercontent.com/6248794/147512939-5bc5ef65-9652-4089-ac98-7f35bf46fbe1.png)
